client/gutenberg/editor/calypsoify-iframe.jsx

```diff
@@ -0,0 +1,62 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteOption, getSiteAdminUrl } from 'state/sites/selectors';
+import { addQueryArgs } from 'lib/route';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const getIframeUrl = ( siteAdminUrl, postId, postType ) => {
+	if ( postId ) {
+		return `${ siteAdminUrl }post.php?post=${ postId }&action=edit&calypsoify=1`;
+	}
+	if ( 'post' === postType ) {
+		return `${ siteAdminUrl }post-new.php?calypsoify=1`;
+	}
+	return `${ siteAdminUrl }post-new.php?post_type=${ postType }&calypsoify=1`;
+};
+
+class CalypsoifyIframe extends Component {
```

Comment:
for those 👀: we could use a stateless component, but we'll need local state and lifecycle methods down the road to support `pub/sub` with the iframe.